design/KruizeLocalAPI.md

```diff
@@ -2745,13 +2745,42 @@ If no experiment type is specified, it will default to `container`.
 
 **Request with `model_settings` and `term_settings` field**
 
-Under `recommendation_settings`, the `model_settings` and `term_settings` fields are optional 
-but can be used to specify model and term details. Currently, for monitoring mode user can set
-model as cost and/or performance and terms can be set to short, medium and/or long term.
-By default, it provides recommendations for all three terms and both models.
+Kruize provides both cost and performance recommendations by default,
+but users often have specific optimization goals. For example, if you're only focused on performance, 
+showing cost-related suggestions can lead to confusion and clutter.
 
-If mode is set to auto or recreate then there can only be one term and one model chosen.
-By default, model will be `performance` and term will be set to `short` term.
+So to streamline the user experience, you can now customize the recommendation_settings in your YAML to specify:
+- Models: performance, cost, or a custom model using CPU, memory, and GPU percentiles
+- Terms: short_term, medium_term, or long_term
+
+In monitoring mode, multiple models and terms are supported. By default, it provides recommendations for both models across all three terms.
+In auto or recreate mode, only one model and one term can be set—defaulting to performance and short_term if omitted.
+
+#### How to specify your Custom Model
+
+Under model_settings:
+
+- "models": ["balance"] defines the name of your custom model. This list can have multiple model names as well.
+- "model_tunables" specifies how the model behaves.  For instance, in this example, the model prioritizes CPU usage at 92%, memory at 89%, and GPU (accelerator) usage at 100% when generating recommendations.
```

Comment:
Those are not percentages but percentiles. I would word it as 
`For instance, in this example, recommendations will be generated based on the 92nd percentile for CPU, 89th percentile for Memory and the 100th percentile for Accelerators`